### ISSUE TITLE
Feature Request: Quick Access to Remove File From Library & Disk

### ISSUE THREAD

**Issue by viking2** [bkury/skin.night#23]
I frequently delete watched files from the library and disk using the remote.

To get to the context menu, I need to press the OK button for a few seconds. However, sometimes this fails and it just plays the file instead (and I need to restart Kodi to get it to work again).

I wonder if you could assign the right arrow to go to the context menu (like in Plex OpenPHT)? Now it just shows the episode number and "Page 1/1"

**However, when it does work, it is still cumbersome:**
1. Long press OK to go to the context menu
2. Scroll down to Manage & Select it
3. Select Manage
4. Scroll to Remove From Library & select it.
5. Press Yes to remove from library
6. Press Yes to remove file

Even better would be to assign the right arrow to go to directly to the Remove From Library item or at least to the Manage item.

**Comment by bkury** [comment]
> I wonder if you could assign the right arrow to go to the context menu (like in Plex OpenPHT)? Now it just shows the episode number and "Page 1/1"

Unfortunately, this idea would be very inconsistent. Only the "List" viewmodes would actually work with this (Selected Item --> Left --> ContextMenu for selected item) but this wouldn't work with any other viewmode because left/right navigation could be another item.

In your case, I would assign the "Delete" function directly to a remote button. I don't know what OS or remote you are using but you could take a look at your keyboard.xml or remote.xml KODI configuration file. 

I can assist you with this change if you give me some more information like OS, Remote model, ...

**Comment by viking2** [comment]
Thank you for offering to help :-) !!
I am using WIN7 (Ultimate; 64 bit), Kodi 18.9 and Windows Media Center RC6 RRS9002 Remote Control 
![](https://i.gyazo.com/b8cbf7ed357e03b8626ed35935b1e10f.png)

I am using the MCERemote Kodi Add-on and have assigned the Red/Orange button to start Kodi and put Kodi back in focus (ctrl-shift-L = icon shortcut; although it often doesn't work well!) and the Green button to switch Kodi between my two monitors (win-shift-right; Kodi is on my second monitor in a different room from the PC and primary monitor).

BTW, is there maybe some way to use the keyboard.xml or remote.xml KODI configuration files and assign a button directly to the path for kodi.exe rather than the icon shortcut (ctrl-shift-L)? Alternatively, is there some command to bring Kodi into focus (it is often lost when some window pops up when watching)

**Comment by bkury** [comment]
I don't have a MCE remote to test with but this should work:

MCERemote Kodi Add-on --> Configure --> Button Mappings

- Select a button and assign the function "Delete"
- I would also suggest that you assign a dedicated button for function "ContextMenu"

Now you should be able to delete media files and open the context menu with a press of a button.

>Alternatively, is there some command to bring Kodi into focus (it is often lost when some window pops up when watching)

I'm not sure about that. A quick google search pointed me to the following script but I have no idea if this still works with win10:
 https://stackoverflow.com/questions/14760518/how-can-i-maximize-a-specific-window-through-cmd-windows

**Comment by viking2** [comment]
It was simple to just add a delete command. That seems to work. Thanks!

I executed the batch file in your link, to try to maximize Notepad, but it didn't work. 
However, the script here (the last one) seems to work when executing the batch file from windows:
https://stackoverflow.com/questions/35988863/using-command-line-batch-to-switch-to-focus-on-app

I am not sure how to execute this from the remote though. I looked at the keymap here:
https://kodi.wiki/view/Keymap#Universal_Remotes 
I found this command:
System.Exec(exec)
where exec is the path to the batch file.
I added this to one of the buttons on the remote (using the Add-on) but that didn't work....
Any ideas how to execute the batch file? If not, I guess I could ask the author of the MCERemote Add-on...


p.s. I am still using WIN7..
